src/components/PriceDisplay/PriceDisplay.js

```diff
@@ -27,7 +28,8 @@ const Top = styled.div`
 
 const Bottom = styled.div`
 	display: flex;
-	justify-content: flex-end;
+	justify-content: ${({ timerShown = false }) =>
```

Comment:
boolean values generally should follow a`verbNoun` pattern
ie: `showTimer`, `hasTimer`, `displayTimer`, etc.